src/Collections-Unordered/Bag.class.st

```diff
@@ -171,7 +171,16 @@ Bag >> includes: anObject [
 ]
 
 { #category : #enumerating }
-Bag >> keysAndValuesDo: aTwoArgBlock [
+Bag >> keys [
+	"Returns the keys of the receiver."
+	
+	"Bag new add: #a; add: #a; add: #b; keys >>> #(#a #b)"
```

Comment:
This doc comment is missing a parenthesis and seems to be breaking the test run in the CI